src/coreclr/jit/gentree.cpp

```diff
@@ -25502,13 +25515,38 @@ genTreeOps GenTreeHWIntrinsic::HWOperGet()
         case NI_SSE2_Xor:
         case NI_AVX_Xor:
         case NI_AVX2_Xor:
+        case NI_AVX512F_Xor:
+        case NI_AVX512DQ_Xor:
 #elif defined(TARGET_ARM64)
         case NI_AdvSimd_Xor:
 #endif
         {
             return GT_XOR;
         }
 
+#if defined(TARGET_XARCH)
+        case NI_SSE_Or:
+        case NI_SSE2_Or:
+        case NI_AVX_Or:
+        case NI_AVX2_Or:
+        case NI_AVX512F_Or:
+        case NI_AVX512DQ_Or:
+#endif
```

Comment:
We need to either include the `{ return GT_OR; }` as part of the `ifdef` -or- more ideally we add the relevant:
```
#elif defined(TARGET_ARM64)
    case NI_AdvSimd_Or:
#endif
```